### ISSUE TITLE
Doesn't handle two types with the same name in the same assembly.

### ISSUE THREAD

**Issue by grokys** [SuperJMN/OmniXAML#91]
tldr; OmniXaml doesn't respect namespaces when looking up a type.

In the `Perspex.SceneGraph` assembly, we have an immutable `Perspex.Media.SolidColorBrush`. To get round the fact that XAML doesn't work with immutable types, I've also defined a `Perspex.Media.Mutable.SolidColorBrush` class. The `Perspex.Media.Mutable` namespace appears in the `https://github.com/perspex/mutable` xml namespace whereas `Perspex.Media.SolidColorBrush` is in the `https://github.com/perspex` namespace.

However, OmniXAML doesn't respect the defined namespaces. The following XAML tries to create an instance of the *immutable* `SolidColorBrush`:

```
<Style xmlns="https://github.com/perspex"
       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
       xmlns:mut="https://github.com/perspex/mutable">
  <Style.Resources>
    <mut:SolidColorBrush x:Key="Foo">#ff808080</<mut:SolidColorBrush>
  </Style.Resources>
</Style>
```
